### ISSUE TITLE
add attribute filter

### ISSUE THREAD

**Issue by davelandry** [cnavarreteliz/vizbuilder#40]
for example, if selecting "show me occupations", a filter should appear that says "filter by location", where you are able to select from a list of all available occupations

sometimes there may be multiple filter types available, so the interface should have an "add a filter" button, where they then select "location" from a dropdown of available filteres

**Comment by davelandry** [comment]
@frabarz I like the popover to handle the filter configuration, but there are 2 different types of entities we could use for filters: **dimensions** and **measures**.

Currently, you have implemented filtering by measures, which looks great. I would remove the "show/don't show" dropdown, and just have the interface look like this:

---

At first, only show one dropdown combining all of the available measures and dimensions.

```js
Filter by: [ select a dimension/measure ]
```
For example, a user could select a measure like "Average Wage" or a dimension like "Location" from this single dropdown. If they have selected a measure, you would make this interface appear underneath the "Filter by" dropdown:

```js
[ select EQUAL/LOWER/etc ] [ input number box ]
```

And if they have selected a dimension, you would show a dropdown for them to select the new dimension:

```js
[ select a Location ]
```

To summarize, if they want to filter by all salaries lower than 100,000, they would see this in the end:

```js
Filter by: [ Salary ]
[ LOWER ] [ 100000 ]
```

And if they want to filter by only things inside of Concepción:

```js
Filter by: [ Location ]
[ Concepción ]
```

**Comment by davelandry** [comment]
Let's implement the same behavior as in the Mondrian-Rest-UI "cut" interface: https://chilecube.datawheel.us/ui

It simply loads the attribute list into a selector. I know we discussed how creating a giant list is sometimes very intense on the DOM, but let's start here, and then potentially switch to a React component that handles large lists gracefully. Here is @jazzido's logic for fetching the cut data: https://github.com/jazzido/mondrian-rest-ui/blob/master/src/redux/reducers/cutModal.js#L33-L45

**Comment by davelandry** [comment]
@frabarz functionality looks working to me. can we close this issue?

**Comment by frabarz** [comment]
Yeah, it's better to open new issues for further additions.
